### ISSUE TITLE
[SUGGESTION] Building a release with recent fixes and RDS certificates updates

### ISSUE THREAD

**Issue by juno-visualsquares** [sidorares/node-mysql2#1037]
I see the recent releases `2.0.0-alpha1` contain major changes in API 
```
MAJOR: new authPlugins api replacing authSwitchHandler, added caching_sha2_password and mysql_native_password as default plugins. Added tests for mysql 8 and ssl. Mysql 8 server now supported with default settings. #1021, #906, #991
MAJOR: LOCAL INFILE does not automatically read #1034 from the local fs and now requires an explicit infileStreamFactory to be specified in query options.
```
as well as fixes & SSL profie for AWS

```
Update to 2019 CA Amazon RDS certificates #1032 
Update SSL Profile for AWS Serverless Aurora #1026
fix pool ignoring namedPlaceholders config #1022
```

Bundling these as an alpha release may not be a good choice, as users might need non-trivial works porting the usages of authSwitchHandler etc. 
And for those working with AWS env, they might just want the non-MAJOR changes here. (Considering the RDS CA cert upgrade deadline would be soon, I guess there may be others need this as well)

Would it be considered to just make a release for that?

**Comment by tfrancois** [comment]
Hi any update on the SSL certificates update for AWS.  The current AWS SSL certs expire next month (Feb 2020).  Please advise.

**Comment by sidorares** [comment]
@tfrancois from latest related comments I believe certs are up to date. Can you double check this ( and if not - what is missing and what needs to be updated? )

**Comment by tfrancois** [comment]
@sidorares  You're absolutely right.  My mistake...I confirmed this is not an issue as connecting to my AWS DB with updated certs worked with no issue.  My fault for not confirming this before I posted here.  Thank you sir!
